compiler/src/iree/compiler/Codegen/SPIRV/SPIRVMaterializeExecutableConditions.cpp

```diff
@@ -219,6 +233,9 @@ void buildDeviceQueryRegion(const KernelFeatures &features, Value device,
   if (features.coopMatrix) {
     result = buildQueryOp("coopmatrix.ops", features.coopMatrix, result);
   }
+  if (features.address) {
+    result = buildQueryOp("address.mode", features.address, result);
```

Comment:
I think normally we call this addressing model instead of address mode? So I think should be `address.model` here.

Reply:
I believe there are two similar terms: addressing mode and memory model
* https://en.wikipedia.org/wiki/Addressing_mode#:~:text=Addressing%20modes%20are%20an%20aspect,(s)%20of%20each%20instruction.
* https://en.wikipedia.org/wiki/Memory_model_(programming)

I think what we have in this context is more similar to an addressing mode

Reply:
Yeah. I guess it's really not unified. I'm fine with it then.